index.html

```diff
@@ -1,18 +1,17 @@
 <!DOCTYPE html>
```

Comment:
- [ ] You have done a good job with the accessibility :smiley: but I would suggest updating the `alt` attribute for the skills images because all show "javascript" when there are other technologies :wink:
![image](https://user-images.githubusercontent.com/81522435/144903668-571da1fd-dc4f-4d87-b471-98cf28d36830.png)

Reply:
Done